frontend/components/FileUploadMedia/FileUploadMedia.jsx

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+import {h, Component} from 'preact'
+import proptypes from 'proptypes'
+
+import {bindActionCreators} from 'redux'
+import {connectHelper} from 'lib/util'
+
+import Style from 'lib/Style'
+import styles from './FileUploadMedia.css'
+
+import * as documentActions from 'actions/documentActions'
+import * as documentsActions from 'actions/documentsActions'
+
+import DropArea from 'components/DropArea/DropArea'
+import FileUpload from 'components/FileUpload/FileUpload'
+
+class FileUploadMedia extends Component {
+
+  static propTypes = {
+    /** 
+     * The global actions object
+     */
+    actions: proptypes.object,
+    /**
+     * The current bucket object
+     */
+    bucket: proptypes.object,
+  }
+
+  render() {
+    return (
+      <div class={styles.placeholder}>
+        <div class={styles['upload-options']}>
+          <DropArea
+            draggingText={`Drop image(s) here`}
+            onDrop={this.handleFileChange.bind(this)}
+          >
+            <div class={styles['upload-drop']}>
+              Drop file(s) to upload
+            </div>
+          </DropArea>
+        </div>
```

Comment:
```suggestion
        </div>

```

Reply:
Huh? What does this mean? Do you mean we should have empty lines at the end of each file?

Reply:
This is GitHub being weird. I was just suggesting a blank line between components.